translate/PT.json

```diff
@@ -422,30 +422,30 @@
         ["rating.limitsame", "Você não pode alterar a classificação do mesmo jogador 2 dias seguidos."],
         ["rating.limitday", "Você só pode mudar a classificação uma vez por dia."],
         ["rating.limitself", "Você não pode mudar sua própria classificação."],
-        ["scoreboard.title", "Online: %s\/%s\nPlaying: %s"],
+        ["scoreboard.title", "Online: %s\/%s\nJogando: %s"],
         ["scoreboard.name", "Nome"],
         ["scoreboard.murders", "Matou"],
         ["scoreboard.lifetime", "Tempo vivo"],
-        ["scoreboard.ping", "Ping"],
+        ["scoreboard.ping", "Latência"],
```

Comment:
![image](https://github.com/salwador/MTA-DayZ-Ultimate/assets/12719219/f9028573-56be-4c47-bb58-978f06caa43d)
Im think dont need translate "Ping" word, because it's more technical info, and is unlikely to make the user experience better.